test/parallel/test-bootstrap-modules.js

```diff
@@ -9,7 +9,7 @@ const common = require('../common');
 const assert = require('assert');
 
 const isMainThread = common.isMainThread;
-const kMaxModuleCount = isMainThread ? 56 : 78;
+const kMaxModuleCount = isMainThread ? 58 : 80;
```

Comment:
This increases the number of module being loaded during bootstrap a little bit, but my hope is to resolve the TODO I added in `setupGlobalConsole` later so that we don't wrap the console with inspector APIs when inspector is not even activated (I believe it used to be that way, but somehow the lazy-loading got lost during refactoring? cc @eugeneo ).